### ISSUE TITLE
A Few Issues

### ISSUE THREAD

**Issue by PorkHat** [HabitRPG/habitica#90]
First of all I'm logged in on Chrome and I used Facebook to log in.

I started using the ap yesterday and went through the crash. Lost everything. So at about 10:30 PM last night I set it up again.

This morning though

1.) My dailies I checked off last night didn't reset.

2.) Last night when I would check something of the todo list it wouldn't disapear. Sometimes the line item would be duplicated. When I deleted it there would be a blank item.

3.) I actually am not upset about #3 but know it's a bug so I'm reporting it anyway. My first item under habits can only be plussed, there is no minus icon there as I removed it. It's still being charted and I can click the progress icon to look at it. 

Thanks.

**Comment by lefnire** [comment]
noted. will get to this in a few days, #91 will occupy today

**Comment by AquaTsubaki** [comment]
I had a similar issue.  I am also in Chrome, but am logging in with the HabitRPG username and password I created.  I have been using HabitRPG for about a week now.  

When I woke up this morning I also had problem (1) (my dailies I checked off last night did not reset).  I left them checked off and did not uncheck them.  At some point doing the afternoon they came unchecked (I was offline from about 12:30AM-6:30PM), but if I look at my chart it is graphed as though I completed them today.

Thanks!
